cmd/generate-signature/main.go

```diff
@@ -41,25 +42,25 @@ func main() {
 		os.Exit(1)
 	}
 
-	pathToClientCert := os.Getenv("CF_INSTANCE_CERT")
-	pathToClientKey := os.Getenv("CF_INSTANCE_KEY")
+	pathToInstanceCert := os.Getenv("CF_INSTANCE_CERT")
+	pathToInstanceKey := os.Getenv("CF_INSTANCE_KEY")
 	roleName := os.Getenv("ROLE")
 
-	clientCertBytes, err := ioutil.ReadFile(pathToClientCert)
+	instanceCertBytes, err := ioutil.ReadFile(pathToInstanceCert)
 	if err != nil {
-		fmt.Println(err)
+		log.Fatal(err)
```

Comment:
I simply updated this to `log` in multiple places to standardize the type of output given by the tools in this repo.